python/src/array.cpp

```diff
@@ -562,6 +562,14 @@ void init_array(py::module_& m) {
           Returns:
             list(int): A list containing the sizes of each dimension.
         )pbdoc")
+      .def_property_readonly(
+          "strides",
+          [](const array& a) { return a.strides(); },
```

Comment:
The strides are an implementation detail in MLX which should not be exposed to the operations. The reason is that until the array is evaluated we cannot know the strides or more generally its memory layout.